examples/meta/src/meta_api/calls.sg

```diff
@@ -1,6 +1,3 @@
-# static call
-Math:init_random(1)
-
 # global function call
```

Comment:
there is a great side-effect of removing the only source of static calls in the examples: R interfaces did not work with static calls, but that was the only thing, so after this is merged, we can try activating R again :)